src/products/products.model.ts

```diff
@@ -17,9 +17,3 @@ export class Product {
 
 export const ProductSchema = SchemaFactory.createForClass(Product);
 
-export interface Product {
-  id: string;
-  name: string;
-  weight: number;
-  imageUrl: string;
-}
```

Comment:
Again? You did that yesterday as far as I remember. I guess your branch is not up-to-date. Try:

```
git pull origin master
```